newshour-namedentity-wikipedialink/process.py

```diff
@@ -1,9 +1,10 @@
-"""Processes Named Entity Linking annotation files
+"""Processes Named Entity Linking annotation files:
 
-Reads in the '.tab' annotations file containing grounding information in the `BATCH_DIR` directory,
-then reads in each brat .ann file downloaded from URL specified in `base-ner.url` file in the BATCH_DIR
-Fetches wikidata QIDs using the wikipedia URLs in the data.
-Generates a tsv file for each unique GUID. Exports the results to a "golds" directory
+Reads in the NEL `.tab` annotations file containing grounding/linking information in the `BATCH` directory,
```

Comment:
In process.py, for this line, added similar wording for a little redundancy to tie which file types are from which datasets. 
As shown, the readme does explain grounding equalling linking.